Matlab_Scripts/5_Depth_Colourless/Study/master.m

```diff
@@ -0,0 +1,17 @@
+% This script either runs preprocesses or analysis, for the appropriate
```

Comment:
again lets save making these in the study folder till we agree on the pilot folder